### PR TITLE
Add Sponsorships page, with list of sponsors and information for how to offer a sponsorship

### PR DESCRIPTION
**Pull request by Southpaw1496**
I'm not entirely happy with the formatting yet, I thought it might be nice to have each sponsor's logo next to their entry, but I don't know how to do that elegantly.

### CODE REVIEW

**Comment by gdude2002** [comment]
"Past Sponsors" makes no sense imo, the point of a sponsorship for many sponsors is we're sending people their way - doesn't really make sense to do that if they're no longer sponsoring us.

**Comment by Southpaw1496** [comment]
> "Past Sponsors" makes no sense imo, the point of a sponsorship for many sponsors is we're sending people their way - doesn't really make sense to do that if they're no longer sponsoring us.

As I mentioned in Discord, this page is purely for informational purposes, it's not designed to be an advertisement for the sponsors, which is why Endorsed and non-Endorsed sponsors get equal preference. The advertisement part would go on the homepage and be addressed separately (#68

**Comment by Southpaw1496** [comment]
@Akarys42 @Redblueflame Your input would also be appreciated, as the other two people most interested in the sponsorship process. I'm also requesting @pluiedev 's review since I don't really know what I'm doing in terms of Astro yet 🙃

**Comment by Redblueflame** [comment]
Addressing @gdude2002's comment:
Getting a list of past sponsorships is worth, but we have to make sure that we only put names, not more (link to website ok, but not encouraged)

For that, I think putting past memberships below the fold by putting "How do I sponsor" just above can be a good placement

**Comment by Southpaw1496** [comment]
I mean, I don't think a link to the website is problematic, especially as the page is for information and also hidden away in the sidebar. Especially when Endorsed sponsors have their own place on the homepage, I don't think most people will go hunting for that page.

**Comment by Southpaw1496** [comment]
I do think we should be keeping the list of sponsors together, but putting them below the "How to be a sponsor" section would be OK if you feel strongly about it.

File: src/pages/en/sponsors.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+There are two main types of sponsors, endorsed and non-endorsed. Endorsed sponsors are those that require an endorsement from us, such as their logo on our site homepage. Non-endorsed sponsors do not require an endorsement.
+
+## Current Sponsors
+
+### 1Password (Non-Endorsed)
+1Password is a company that makes password managers and other tools for securely storing credentials. They provide us with a free 1Password Teams account to share infrastructure keys, social media logins, and other sensitive data, as part of their [1Password for Open Source Projects](https://github.com/1Password/1password-teams-open-source) program.
+
+### Starchild Systems (Endorsed)
+Starchild Systems is an Infrastructure-as-a-Service company. They host the [Quilt Forum](https://forum.quiltmc.org) on their infrastructure at no cost to us.
```
Comment (pluiedev) [inline src/pages/en/sponsors.mdx:19]: The forum link should reuse the URLs set in `Site.ts`.
```mdx
---
# ...
---
import site from "@data/Site";

...

### Starchild Systems (Endorsed)
Starchild Systems is an Infrastructure-as-a-Service company. They host the <a href={site.forum}>Quilt Forum</a> on their infrastructure at no cost to us.
```
We should also add the OpenCollective page to `Site.ts` at some point, but I wouldn't force you to do it :>

**Review by pluiedev — CHANGES_REQUESTED** [review]
Mostly good tech-wise, just a couple nitpicks

**Review by pluiedev — APPROVED** [review]

**Review by gdude2002 — APPROVED** [review]

**Pull request merged**